lib/src/store.dart

```diff
@@ -135,7 +138,6 @@ class Store extends Disposable {
 
     StreamSubscription<Store> sub = _stream.listen(onData,
         onError: onError, onDone: onDone, cancelOnError: cancelOnError);
-    manageStreamSubscription(sub);
     return sub;
```

Comment:
Could we go back to returning `_stream.listen()` directly now since we're not doing anything with the subscription? :)